auth/config.go

```diff
@@ -147,6 +147,39 @@ func SetContextProject(kubeconfigPath string, contextName string, project string
 	return clientcmd.WriteToFile(*kubeconfig, kubeconfigPath)
 }
 
+// SetContextOrganization sets the given organization in the given context of the kubeconfig
+func SetContextOrganization(kubeconfigPath string, contextName string, organization string) error {
+	kubeconfig, err := clientcmd.LoadFromFile(kubeconfigPath)
+	if err != nil {
+		return fmt.Errorf("kubeconfig not found: %w", err)
+	}
+	context, exists := kubeconfig.Contexts[contextName]
+	if !exists {
+		return fmt.Errorf("could not find cluster %q in kubeconfig", contextName)
+	}
+	extension, exists := context.Extensions[util.NctlName]
+	if !exists {
+		return ErrConfigNotFound
+	}
+
+	cfg, err := parseConfig(extension)
+	if err != nil {
+		return err
+	}
+
+	cfg.Organization = organization
```

Comment:
Maybe we could exit here already if the organization which we want to set is already set? That way we wouldn't overwrite the `context.Namespace` later if the user already set this to be something else.

Reply:
yeah - very good idea! 👌